src/include/rbd/librbd.hpp

```diff
@@ -241,6 +241,9 @@ class CEPH_RBD_API Image
   int is_exclusive_lock_owner(bool *is_owner);
   int lock_acquire(rbd_lock_mode_t lock_mode);
   int lock_release();
+  int lock_get_owners(rbd_lock_mode_t *lock_mode,
+                      std::list<std::string> *lock_owners);
+  int lock_break(rbd_lock_mode_t lock_mode, const std::string &lock_owner);
```

Comment:
I can build a clustering-away application with multiple instances, each using librbd shared locks. If my management layer decides to STONITH one of the shared lock owners, it should be possible.